game/scripts/npc/abilities/nyx_assassin_innate_mana_burn_oaa.txt

```diff
@@ -0,0 +1,26 @@
+"DOTAAbilities"
+{
+  //=================================================================================================================
+  // Nyx Assassin: Innate Mana Burn
+  //=================================================================================================================
+  "nyx_assassin_innate_mana_burn_oaa"
+  {
+    // General
+    //-------------------------------------------------------------------------------------------------------------
+    "BaseClass"                                           "ability_lua"
+    "ScriptFile"                                          "abilities/oaa_nyx_assassin_innate_mana_burn.lua"
+    "AbilityBehavior"                                     "DOTA_ABILITY_BEHAVIOR_PASSIVE | DOTA_ABILITY_BEHAVIOR_HIDDEN"
+    "SpellDispellableType"                                "SPELL_DISPELLABLE_NO"
+
+    "MaxLevel"                                            "6"
+    "RequiredLevel"                                       "-1"
+
+    "DependentOnAbility"                                  "nyx_assassin_vendetta"
+
+    "AbilityValues"
+    {
+      "mana_pct"                                          "13 17 21 25 29 33"
```

Comment:
it's current, not max, but still maybe too high

Reply:
13 15 17 19 21 23 might even be too high